sites/public/src/lib/hooks.ts

```diff
@@ -57,6 +57,10 @@ export const useGetApplicationStatusProps = (listing: Listing): ApplicationStatu
   return props
 }
 
+// These functions were sparsely used and/or completely ignored, so I'm commenting
+// them out.  If it turns out they are needed later then it should be easy to add
+// them back by uncommenting, but make sure the logic still matches expectations.
+/*
```

Comment:
Most of these were either only used in one place or stopped being used several PRs ago.  Either way, the last remaining bit of functionality was aggregated into the `ListingService` class to provide better grouping of logic and functionality.  They aren't 1:1 with the analogous methods in that class though, in part due to the fact that searching on the `/listings/combined` endpoint uses different classes/enums than the `/listings` endpoint.  Rather than remove them completely, I commented them out _just in case_